x-pack/test/functional/apps/api_keys/home_page.ts

```diff
@@ -51,6 +65,10 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
         await pageObjects.apiKeys.deleteAllApiKeyOneByOne();
       });
 
+      after(async () => {
+        await clearAllApiKeys();
```

Comment:
if it is placed here, do we still need it in `before` hook?

Reply:
Yes, because before the first test runs, some other apps drop in some API Keys so if the first test runs, it may fail. That's why the test was failing in the first place.